common/source/docs/common-uavcan-escs.rst

```diff
@@ -79,6 +81,7 @@ The RCOU messages are also written to the onboard logs which hold the requested
 Additional information
 ======================
 
+:ref:`Hobbywing CAN ESCs <common-hobbywing-dronecan-esc>`
```

Comment:
you could put the toctree here

Reply:
I think I'll remove this link.  It's higher up in the page and we've agreed to leave it on the motors-and-escs landing page.